src/dataframe/dataframe.jl

```diff
@@ -596,10 +596,34 @@ Base.setindex!(df::DataFrame, x::Void, col_ind::Int) = delete!(df, col_ind)
 
 Base.empty!(df::DataFrame) = (empty!(df.columns); empty!(index(df)); df)
 
-function Base.insert!(df::DataFrame, col_ind::Int, item::AbstractVector, name::Symbol)
+# TODO: add docstring of make_unique after 0.4 release of DataFrames
```

Comment:
0.4? And why not document `make_unique` immediately? It's also fine to avoid documenting deprecated arguments.

Reply:
What release number should it be then? I will try to add documentations (although this is massive and I might miss something).

Reply:
I wouldn't mention the name of the version here, it's easy to find out the history from git anyway.